### PR TITLE
More traitor to contractor renaming

### PR DESCRIPTION
**Pull request by Iglpigl**
## About The Pull Request

Changes more traitor text to contractor.

## Why It's Good For The Game 

If have any questions read uplink update document in mines. If there are still questions left - go to fortune master, aka rope bunny.

## Changelog
:cl:
spellcheck: changes mentions of traitor into contractor 
:cl:

### CODE REVIEW

**Review by TheShown911 — COMMENTED** [review]
Changing code comments is pretty much useless when variable names and others still mention traitors, either change everything or only the things players can see

**Comment by Iglpigl** [comment]
> Changing code comments is pretty much useless when variable names and others still mention traitors, either change everything or only the things players can see

Problem addressed the codebase is now rewritten.

File: code/__DEFINES/gamemode.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 // antag bantypes macros.
 #define ROLE_BANTYPE_BORER ROLE_BORER
 #define ROLE_BANTYPE_MALFUNCTION ROLE_MALFUNCTION
-#define ROLE_BANTYPE_TRAITOR ROLE_TRAITOR
+#define ROLE_BANTYPE_CONTRACTOR ROLE_CONTRACTOR
```
Comment (TheShown911) [inline code/__DEFINES/gamemode.dm:35]: Changing this will probably remove all the traitor bans from everyone.

Comment (Kegdo) [reply code/__DEFINES/gamemode.dm:35]: __Good.__

Comment (Iglpigl) [reply code/__DEFINES/gamemode.dm:35]: That will only Impact the Russian community so I don't see it as a problem.

File: code/datums/uplink/ammunition.dm [hunk 2]
```diff
@@ -11,70 +11,72 @@
...
 	desc = "SMG .40 magazine, loaded with lethal ammunition. Can fit 25 bullets."
 	item_cost = 3
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
 	path = /obj/item/ammo_magazine/msmg
 
 /datum/uplink_item/item/ammo/magnum/msmg/hv
 	name = "SMG .40 HV magazine"
 	desc = "SMG .40 magazine, loaded with high velocity ammunition. Can fit 25 bullets."
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:76]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 3]
```diff
@@ -11,70 +11,72 @@
...
 	item_cost = 5
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:79]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 4]
```diff
@@ -83,29 +85,38 @@
 	name = ".35 Auto SL box"
 	desc = "Contains 2 standard .35 Auto speed loaders, loaded with lethal ammunition. Can fit 6 bullets."
 	item_cost = 1
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slpistol
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:88]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 5]
```diff
@@ -83,29 +85,38 @@
...
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slpistol
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:91]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 6]
```diff
@@ -83,29 +85,38 @@
...
 
 /datum/uplink_item/item/ammo/slpistol/highvelocity
 	name = ".35 Auto HV SL"
 	desc = "Contains 2 standard .35 Auto speed loaders, loaded with high-velocity ammunition. Can fit 6 bullets."
 	item_cost = 2
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slpistol/hv
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:97]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 7]
```diff
@@ -83,29 +85,38 @@
...
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slpistol/hv
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:100]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 8]
```diff
@@ -83,29 +85,38 @@
...
 
 /datum/uplink_item/item/ammo/slmagnum
 	name = ".40 magnum SL box"
 	desc = "Contains 2 .40 magnum speed loaders, loaded with lethal ammunition. Can fit 6 bullets."
 	item_cost = 1
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slmagnum
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:106]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 9]
```diff
@@ -83,29 +85,38 @@
...
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slmagnum
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:109]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 10]
```diff
@@ -83,29 +85,38 @@
...
 
 /datum/uplink_item/item/ammo/slmagnum/highvelocity
 	name = ".40 magnum HV SL box"
 	desc = "Contains 2 .40 magnum HV speed loaders, loaded with high velocity ammunition. Can fit 6 bullets."
 	item_cost = 4
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slmagnum/highvelocity
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:115]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 11]
```diff
@@ -83,29 +85,38 @@
...
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slmagnum/highvelocity
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:118]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 12]
```diff
@@ -83,29 +85,38 @@
...
+
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:119]:
```suggestion
```

File: code/datums/uplink/ammunition.dm [hunk 13]
```diff
@@ -83,29 +83,30 @@
 	name = ".35 Auto SL box"
 	desc = "Contains 2 standard .35 Auto speed loaders, loaded with lethal ammunition. Can fit 6 bullets."
 	item_cost = 1
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slpistol
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slpistol
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:87]:
```suggestion
	path = /obj/item/storage/box/syndie_kit/slpistol
```

File: code/datums/uplink/ammunition.dm [hunk 14]
```diff
@@ -83,29 +83,30 @@
...
 
 /datum/uplink_item/item/ammo/slpistol/highvelocity
 	name = ".35 Auto HV SL"
 	desc = "Contains 2 standard .35 Auto speed loaders, loaded with high-velocity ammunition. Can fit 6 bullets."
 	item_cost = 2
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slpistol/hv
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slpistol/hv
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:94]:
```suggestion
	path = /obj/item/storage/box/syndie_kit/slpistol/hv
```

File: code/datums/uplink/ammunition.dm [hunk 15]
```diff
@@ -83,29 +83,30 @@
...
+
 
 /datum/uplink_item/item/ammo/slmagnum
 	name = ".40 magnum SL box"
 	desc = "Contains 2 .40 magnum speed loaders, loaded with lethal ammunition. Can fit 6 bullets."
 	item_cost = 1
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slmagnum
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slmagnum
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:102]:
```suggestion
	path = /obj/item/storage/box/syndie_kit/slmagnum
```

File: code/datums/uplink/ammunition.dm [hunk 16]
```diff
@@ -83,29 +83,30 @@
...
 
 /datum/uplink_item/item/ammo/slmagnum/highvelocity
 	name = ".40 magnum HV SL box"
 	desc = "Contains 2 .40 magnum HV speed loaders, loaded with high velocity ammunition. Can fit 6 bullets."
 	item_cost = 4
-	antag_roles = list(ROLE_TRAITOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
-	path = /obj/item/storage/box/syndie_kit/slmagnum/highvelocity
+	antag_roles = list(ROLE_CONTRACTOR,ROLE_MARSHAL,ROLE_INQUISITOR,ROLE_MERCENARY,ROLE_CARRION)
+	path = /obj/item/ammo_magazine/slmagnum/highvelocity
```
Comment (Iglpigl) [inline code/datums/uplink/ammunition.dm:109]:
```suggestion
	path = /obj/item/storage/box/syndie_kit/slmagnum/highvelocity
```

**Comment by Iglpigl** [comment]
Should be fine now.

**Comment by drexample** [comment]
Merge conflict moment. If not fixed until next friday this PR will be closed.

**Comment by Iglpigl** [comment]
Conflict fixed.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 17]
```diff
@@ -831,7 +831,8 @@
 
 /obj/item/reagent_containers/food/snacks/donkpocket
 	name = "donk-pocket"
-	desc = "The food of choice for the seasoned traitor."
+	desc = "The food of choice for the seasoned contractor."
+
```
Comment (Iglpigl) [inline code/modules/reagents/reagent_containers/food/snacks.dm:835]:
```suggestion
```

File: code/game/antagonist/station/contractor.dm [hunk 18]
```diff
@@ -1,9 +1,9 @@
 // Inherits most of its vars from the base datum.
-/datum/antagonist/traitor
-	id = ROLE_TRAITOR
+/datum/antagonist/contractor
+	id = ROLE_CONTRACTOR
 	protected_jobs = list(JOBS_SECURITY, JOBS_COMMAND)
-	bantype = ROLE_TRAITOR
-	antaghud_indicator = "hudtraitor"
+	bantype = ROLE_CONTRACTOR
+	antaghud_indicator = "hudcontractor"
```
Comment (Firefox13) [inline code/game/antagonist/station/contractor.dm:6]: this is used in .dmi file, if I recall correctly for ghosts antaghud and you didn't change the .dmi

**Review by Firefox13 — APPROVED** [review]
Should be good

**Pull request merged**